roles/base/templates/run-ansible-pull.sh.j2

```diff
@@ -38,26 +39,35 @@ while [[ $RETRY -lt $RETRIES ]]; do
         -i localhost \
         --vault-password-file=/home/{{ service_account }}/.vault-password \
         -l "localhost,$(hostname),127.0.0.1" \
-        local.yml >$PULL_OUTPUT 2>&1
+        local.yml >$ANSIBLE_LOG 2>&1
     RESULT=$?
     set -e
 
     if [[ $RESULT -eq 0 ]]; then
         break
     else
-        echo '==== RETRYING ====' >> $OUTPUT
+        echo '==== RETRYING ====' >> $RETRY_LOG
         sleep 30
     fi
 done
 
-if [ $RESULT -ne 0 ]; then
-    cat $OUTPUT
+if [[ $RESULT -eq 0 ]]; then
+    # If everything was okay, check for deprecations but only at the 00:* run.
+    # These deprecations will happen on every run so only notify once a day.
+    # The -A3 seems to be enough to get all/most of the deprecation warning to
+    # make sense of what to fix.
+    hour=$(date +%H)
```

Comment:
It likely won't be :00 anymore when the run is complete, right?  Perhaps capture this value at the very start of the script instead?

Reply:
Hour, not minute. All of the `ansible-pull` runs are scattered across the hour so unlikely any of them start at :00. There is the risk that something will start at e.g. 00:59 and finish in the next hour, but my guess was the 11:59 run will be used for this instead.

Reply:
Oh, but the variable case changes. Oops.

Reply:
Haha, well, we both made some silly errors here :)